readalongs/static/read-along-1.3.dtd

```diff
@@ -0,0 +1,109 @@
+<!-- VERSION: 1.2 -->
+<!ELEMENT read-along (meta|text|body|div|span|anchor|silence|graphic|p|s|w)*>
+<!ATTLIST read-along
+  use-assets-folder CDATA #IMPLIED
+  href CDATA #IMPLIED
+  audio CDATA #IMPLIED
+  xml:lang CDATA #IMPLIED
+  language CDATA #IMPLIED
+  lang CDATA #IMPLIED
+  version CDATA #IMPLIED
+  xmlns CDATA #IMPLIED>
+
+<!ELEMENT text (body|div|span|anchor|silence|graphic|p|s|w)*>
+<!ATTLIST text
+  xml:lang CDATA #IMPLIED
+  lang CDATA #IMPLIED
+  fallback-langs CDATA #IMPLIED
+  id CDATA #IMPLIED>
+
+<!ELEMENT body (div|span|anchor|silence|graphic|p|s|w)*>
+<!ATTLIST body
+  xml:lang CDATA #IMPLIED
+  lang CDATA #IMPLIED
+  id CDATA #IMPLIED>
+
+<!ELEMENT anchor EMPTY>
+<!ATTLIST anchor time CDATA #REQUIRED>
+
+<!ELEMENT silence EMPTY>
+<!ATTLIST silence dur CDATA #REQUIRED>
+
+<!ELEMENT graphic EMPTY>
+<!ATTLIST graphic
+  url CDATA #REQUIRED
+  id CDATA #IMPLIED>
+
+<!ELEMENT div (#PCDATA|div|span|anchor|silence|graphic|p|s|w)*>
+<!ATTLIST div
+  xml:lang CDATA #IMPLIED
+  lang CDATA #IMPLIED
+  id CDATA #IMPLIED
+  class CDATA #IMPLIED
+  type CDATA #IMPLIED
+  do-not-align CDATA #IMPLIED
+  time CDATA #IMPLIED
+  dur CDATA #IMPLIED>
+
+<!ELEMENT span (#PCDATA|div|span|anchor|silence|graphic|p|s|w)*>
+<!ATTLIST span
+  xml:lang CDATA #IMPLIED
+  lang CDATA #IMPLIED
+  id CDATA #IMPLIED
+  class CDATA #IMPLIED
+  type CDATA #IMPLIED
+  do-not-align CDATA #IMPLIED
+  time CDATA #IMPLIED
+  dur CDATA #IMPLIED>
+
+<!ELEMENT p (#PCDATA|span|anchor|silence|s|w)*>
+<!ATTLIST p
+  xml:lang CDATA #IMPLIED
+  lang CDATA #IMPLIED
+  id CDATA #IMPLIED
+  class CDATA #IMPLIED
+  do-not-align CDATA #IMPLIED
+  time CDATA #IMPLIED
+  dur CDATA #IMPLIED>
+
+<!ELEMENT s (#PCDATA|span|anchor|silence|w)*>
+<!ATTLIST s
+  xml:lang CDATA #IMPLIED
+  lang CDATA #IMPLIED
+  id CDATA #IMPLIED
+  class CDATA #IMPLIED
+  do-not-align CDATA #IMPLIED
+  time CDATA #IMPLIED
+  dur CDATA #IMPLIED
+  annotation-id CDATA #IMPLIED
+  sentence-id CDATA #IMPLIED
+  xmlns CDATA #IMPLIED>
```

Comment:
I'm not sure why we need xmlns here. I get it for the top-level read-along element, and I would get it if every element needed to allow it, but only one additional element I don't get.